### ISSUE TITLE
Microsoft Edge Compatibility

### ISSUE THREAD

**Issue by danieldilly** [euvl/vue-js-toggle-button#16]
This doesn't seem to be working in Microsoft Edge.  Try loading the demo page in it: http://vue-js-toggle-button.yev.io/

**Comment by euvl** [comment]
Hey @Digs3 

Could you please provide more information. 
What is your Edge version?
What is the error in inspector?

**Comment by Digs3** [comment]
Microsoft Edge 38.14393.1066.0 
Microsoft EdgeHTML 14.14393

No errors in console.  The element exists but everything blends in into the background so it's not visible at all.

**Comment by zassq** [comment]
That's because Edge/IE11 don't support CSS Custom Properties yet
It's annoying to debug under Edge/IE...

**Comment by euvl** [comment]
I will add default values for ie

**Comment by bomanden** [comment]
is this updated ?

**Comment by euvl** [comment]
Hi @bomanden, I am afraid I do not have enough time right now to fix it. Wil be happy to review a PR though.

**Comment by richardbishopme** [comment]
Hi @bomanden  and @danieldilly 

I added this to my sass file in an existing project and it appears to work in Edge:

```sass
@supports (-ms-ime-align:auto) {
    .vue-js-switch {
      width: 50px!important;
      height: 22px!important;

      .v-switch-core {

        &:before {
          width: 16px!important;
          height: 16px!important;
        }

        width: 50px !important;
        height: 22px !important;
      }
    }
}
```

@euvl Could this code be acceptable as part of a PR to fix the issue? If so, where in the source do I need to target the code change? Is there anything else I need to be aware of?

**Comment by euvl** [comment]
Hey, looks good!

In Button.vue you can find 

```
.vue-js-switch {
  line-height: var(--h);
  height: var(--h);

  .v-switch-label {
    line-height: var(--h);
    height: var(--h);
  }

  .v-switch-core {
    border-radius: 999px;
    width: 50px;
    width: var(--w);
    height: var(--h);

    &:before {
      width: calc(var(--h) - 6px);
      height: calc(var(--h) - 6px);
    }
  }
```

i think if you will do:

```
height: var(--h);
height: 22px;
```

Browser will ignore the first statement if it is not supported and just use the second ( hence no need to use !important).

If you would form this into a PR, i would be very happy to approve it 🙏 👍

**Comment by euvl** [comment]
@ouraios Has fixed the issue, and it is in master now, will publish it in `v1.1.5` soon.

**Comment by richardbishopme** [comment]
Thanks @ouraios! looks good! Time has been a little against me of late! 

@euvl When I made some changes before, I ran them against your demo in the project. In Edge, I found because of the fixed sizes the components would appear squashed (check the Hodor one) and wouldn't resize based on their content.

I wonder if it is worth allowing a user to specify a width and a height prop? I't won't be a problem for Chrome users, but should allow Edge users to resize if the component displays incorrectly.
